WordPress/Classes/ViewRelated/Post/PostEditorState.swift

```diff
@@ -283,10 +283,17 @@ public class PostEditorStateContext {
     }
 
     /// Returns the WPAnalyticsStat enum to be tracked when this post is published
+    ///
     var publishActionAnalyticsStat: WPAnalyticsStat {
         return editorState.action.publishActionAnalyticsStat
     }
 
+    /// Indicates if the editor should be dismissed whenever the Publish Action is executed
```

Comment:
Consider changing:

`...whenever the Publish Action is executed`

to

`...when the publish button is tapped`